src/commands/key.ts

```diff
@@ -96,30 +112,36 @@ ${cWrap.fb("set")}\t\tset a key. defaults to setting your openAI key
               })
               .example(
                 cWrap.fg("forge key set sk-abcxyz"),
-                cWrap.fc("sets your openAI key")
+                cWrap.fc("sets your provider key")
               ),
           async (args) => {
             const { key, provider } = args;
-            const keyToSet = provider || Keys.OPENAI;
+            const keyToSet =
+              provider || (await selectOption(Object.values(Keys)));
 
-            if (keyToSet === Keys.OPENAI) {
-              const { data, error } = await makeRequest(EP.SET_OPENAI_KEY, {
-                data: { apiKey: key },
+            if ([Keys.OPENAI, Keys.ANTHROPIC].includes(keyToSet)) {
+              const { data, error } = await makeRequest(EP.SET_PROVIDER_KEY, {
+                data: { apiKey: key, provider: keyToSet },
                 method: "POST",
               });
               if (error) {
-                console.log(`${cWrap.br("Error")} setting key.`);
+                console.log(
+                  `\n\n${cWrap.br("Error")} setting key: `,
+                  error?.response?.data?.error
+                );
                 return;
               }
               console.log(
-                `Deployment key successfully updated for ${cWrap.fg(keyToSet)}.`
+                `\n\nDeployment key successfully updated for ${cWrap.fg(
+                  keyToSet
+                )}.`
               );
             }
 
             try {
               localConfigService.storeValue(keyToSet, key);
```

Comment:
key set allows a user to change their forge key in localConfigService but doesn't update their key in DB. Is this expected functionality? Are users going to be allowed to change their forge key?